applications/GeoMechanicsApplication/custom_conditions/micro_climate_constants.h

```diff
@@ -0,0 +1,34 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+#pragma once
+
+namespace Kratos::MicroClimateConstants
+{
+
+constexpr static auto AirDensity = 1.18;
+constexpr static auto AirHeatCapacity = 1004.67;
+constexpr static auto RoughnessLayerResistance = 30.0;
+constexpr static auto LatentEvaporationHeat = 2.45e6;
+constexpr static auto WaterDensity = 1e3;
+constexpr static auto PsychometricConstant = 0.63;
+constexpr static auto SurfaceResistance = 30.0;
+
+constexpr static auto RoughnessLayerHeight = 10.0;
+constexpr static auto VonNeumannCoefficient = 0.4;
+constexpr static auto MeasurementHeight = 10.0;
+constexpr static auto RoughnessHeight = 1.0;
+constexpr static auto GravitationalAcceleration = 9.81;
```

Comment:
This is probably the 4th redefinition of g that we have ( volumetric acceleration process, weights in waterloads etc. ) It really should be only 1 and 1 alone!

Reply:
Completely agreed, however we would like to fix this for all places at once, instead of only for this one (and it's not clear right now where that 1 place would be yet)